internal/repository/memory.go

```diff
@@ -110,6 +110,7 @@ func (n *nodeReaderWriter) URI() fyne.URI {
 // declare conformance with repository types
 var _ repository.Repository = &MemoryRepository{}
```

Comment:
Why in this form:

`var _ repository.Repository = &MemoryRepository{}`

and not this form (seen in widgets)?

```
var _ repository.Repository = (*MemoryRepository)(nil)
```

Reply:
No particular reason. Is there a reason to prefer one over the other?

Reply:
Just consistency/style I think, I do not know if this has an allocation cost difference (it would be negligible if anything.)

Reply:
I can't argue with consistency

Reply:
3b79728860d636ea6a55be83397e0c1e8b5a22c4